addons/l10n_mr/i18n/fr.po

```diff
@@ -0,0 +1,156 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* l10n_mr
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 17.0+e\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2025-01-30 16:22+0000\n"
+"PO-Revision-Date: 2025-01-30 16:22+0000\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: l10n_mr
+#: model:account.report.column,name:l10n_mr.tax_report_tax_16
+msgid "16%"
+msgstr "16%"
+
+#. module: l10n_mr
+#: model:account.report.column,name:l10n_mr.tax_report_tax_18
+msgid "18%"
+msgstr "18%"
+
+#. module: l10n_mr
+#: model:ir.model,name:l10n_mr.model_account_chart_template
+msgid "Account Chart Template"
+msgstr "Modèle de plan comptable"
+
+#. module: l10n_mr
+#: model:account.report.line,name:l10n_mr.tax_report_vat_base
+msgid "Calculation of VAT base"
+msgstr "Calcul de la base de la TVA"
+
+#. module: l10n_mr
+#: model:account.report.line,name:l10n_mr.tax_report_vat_net
+msgid "Calculation of net VAT"
+msgstr "Calcul de la TVA nette"
+
+#. module: l10n_mr
+#: model:account.report.line,name:l10n_mr.tax_report_base_completed_operations
+msgid "Completed Operations"
+msgstr "Opérations Réalisées"
+
+#. module: l10n_mr
+#: model:account.report.line,name:l10n_mr.tax_report_deductions_prev_month
+msgid "Credit carried forward to previous month"
+msgstr "Crédit reporté du mois précédent"
```

Comment:
Either English or French is a bit off. The French says the credit is carried forward **from** the previous month.

Reply:
"Crédit reporté du mois précédent" is correct.